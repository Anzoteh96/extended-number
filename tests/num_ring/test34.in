160775234351951844880970989624239315955898327269855605455230439840792169286587078979715417928620002885
-808321639088392355811662156032952076746958174268484501097247701434796686537175598514406388530780865775
-699412655943416649808505494334389316225891046786834221455260794387279136455667444672903054777725072860
