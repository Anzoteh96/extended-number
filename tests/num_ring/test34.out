Enter the number base b:
Enter two numbers n and m:
b is 160775234351951844880970989624239315955898327269855605455230439840792169286587078979715417928620002885
n mod b is 156329767023318713474163781712483818988431789350649131634134937609956329182346875363886119040939151535
m mod b is 104463515816342574596349453786807263553600589562443805820891404816681709977267950225674034865374941565
n + m is 100018048487709443189542245875051766586134051643237331999795902585845869873027746609844735977694090215
n - m is 51866251206976138877814327925676555434831199788205325813243532793274619205078925138212084175564209970
n * m is 52104296102690328221315831465424371939355591707609993155483316778722628642608840559717858022387711910
n ** m is 6620024234854331842159788856135659806034872977750770006059246168945444826358492294125219648091078560
